lnbits/wallets/lnpay.py

```diff
@@ -69,13 +72,11 @@ async def create_invoice(
         else:
             data["memo"] = memo or ""
 
-        async with httpx.AsyncClient() as client:
-            r = await client.post(
-                f"{self.endpoint}/wallet/{self.wallet_key}/invoice",
-                headers=self.auth,
-                json=data,
-                timeout=60,
-            )
+        r = await self.client.post(
+            "/wallet/{self.wallet_key}/invoice",
```

Comment:
this should be a f string

Reply:
or am i missing something? seems like all over the code?

Reply:
seems like I overlooked those in my workflow. has to be adjusted